backend/src/app/services/associatedSite/associatedSite.service.ts

```diff
@@ -37,7 +37,7 @@ export class AssociatedSiteService {
 
       if (showPending) {
         result = await this.assocSiteRepository.find({
-          where: { siteId, userAction: UserActionEnum.UPDATED },
+          where: { siteId, srAction: SRApprovalStatusEnum.PENDING },
```

Comment:
any related changes required for the unit test ?

Reply:
No unit test were failing due to this.